python/doc/usecases/use_case_oscillator.rst

```diff
@@ -0,0 +1,97 @@
+.. _use-case-oscillator:
+
+A two degree-of-fredom primary/secondary damped oscillator
+==========================================================
+
+We consider a two degree-of-fredom primary/secondary damped oscillator.
+This system is composed of a two-stage oscillator characterized by a mass, a stiffness and a damping ratio for each of the two oscillators.
+This system is submitted to a white-noise excitation.
+The limit-state function is highly non-linear, mainly due to the interactions between the two stages of the system, and presents one failure zone.
+
+
+.. figure:: ../_static/oscillator.png
+    :align: center
+    :alt: use case oscillator
+    :width: 50%
+
+    Two stage oscillator
+
+
+The limit state function is defined as follows:
+
+.. math:: G = F_s - 3 k_s \sqrt{\frac{\pi S_0}{4 \zeta_s \omega_s^3} \left[\frac{\zeta_a \zeta_s}{\zeta_p \zeta_s (4 \zeta_a^2 + \theta^2)+\gamma \zeta_a^2}\frac{(\zeta_p \omega_p^3 + \zeta_s \omega_s^3)\omega_p}{4 \zeta_a \omega_a^4}\right]}
+
+
+The natural frequency of the first oscillator is equal to:
+
+.. math:: \omega_p = \sqrt{\frac{k_p}{m_p}}
+
+The natural frequency of the secondary oscillator is equal to:
+
+.. math:: \omega_s = \sqrt{\frac{k_s}{m_s}}
+
+The average natural frequency of the system is equal to:
+
+.. math:: \omega_a = \frac{\omega_p+\omega_s}{2}
+
+The average damping ratio of the system is equal to:
+
+.. math:: \zeta_a = \frac{\zeta_p+\zeta_s}{2}
+
+The mass ratio is equal to:
+
+.. math:: \gamma = \frac{m_s}{m_p}
+
+The tuning parameter of the system is equal to:
+
+.. math:: \theta = \frac{\omega_p - \omega_s}{\omega_a}
+
+Eight uncertainties are considered in the system:
+- on the masses of the primary and secondary systems (:math:`m_p` and :math:`m_s`),
```

Comment:
The bullet points here are not rendered correctly in the artefact

Reply:
fixed